sa360/src/tests/rules_test.ts

```diff
@@ -176,69 +156,6 @@ export async function generateCampaignPacingTestData(
   }
   return values;
 }
-/**
- *
- * Generates geo test data for the tests below.
- */
-export async function generateUserListTestData(
```

Comment:
Was this unused everywhere?

Reply:
I did a blame and it looks like I wrote the function and it was just never used.